### ISSUE TITLE
Hint for password requirements at README.md

### ISSUE THREAD

**Issue by Cinux90** [while-true-do/ansible-role-nextcloud#3]
### Type of Report

enhancement

### Actual behaviour

Run the role and its failed, because of missing password for database and nextcloud admin.
No hint in the Readme that is should be set.

### Expected behaviour

Hint that you need to set the passwords at README.md
Maybe an switch can be implemented, that the role not runs if this variables are missing. Feel free to start a discussion here :)

**Comment by daniel-wtd** [comment]
Yep, plus it should also be documented in the defaults/main.yml if a variable must be set.
